src/components/vanilla/charts/BubbleMapChart/index.tsx

```diff
@@ -0,0 +1,202 @@
+// React & 3rd Party
+import React, { useEffect, useState } from 'react';
+import { DataResponse, Dataset, DimensionOrMeasure, Measure } from '@embeddable.com/core';
+import Leaflet, { LatLngBoundsExpression, MarkerCluster } from 'leaflet';
+import { MapContainerProps, MarkerProps, TileLayerProps, Tooltip, useMap } from 'react-leaflet';
```

Comment:
"react-leaflet" doesn't include the same imports as the "leaflet" library?

Reply:
Unfortunately, no. See here for info: https://react-leaflet.js.org/docs/start-installation/

Reply:
Oh, hmm, actually maybe - let me see if I can reduce those. You still have to have leaflet in the project but may not need to import from it.

Reply:
@mad-raccoon - confirmed: React-Leaflet does not export those two items.